apiclient/account.go

```diff
@@ -46,7 +46,7 @@ func (c *HTTPclient) Transfer(to common.Address, amount uint64) (types.HexBytes,
 	stx.Tx, err = proto.Marshal(&models.Tx{
 		Payload: &models.Tx_SendTokens{
 			SendTokens: &models.SendTokensTx{
-				Txtype: models.TxType_SET_ACCOUNT_INFO,
+				Txtype: models.TxType_SET_ACCOUNT_INFO_URI,
```

Comment:
Why do we need a set_account_info_uri transaction if we can already set the account using SET_ACCOUNT?